Add dns_enable to variables

ommadawn46 opened the pull request:
## Overview
I add a dns_enable option to variables in this PR.
Thanks to this option, it's possible to build only CloudFront and S3.

## Purpose
Current Route53 setting only creates A record. This PR makes it possible to use customized Route53 module: such as using other DNS record types, setting failover, handling health check, and so on. 
To do that, I add the option to create only CloudFront and S3.

joshmyers commented on main.tf:
```diff
@@ -173,7 +173,7 @@ resource "aws_cloudfront_distribution" "default" {
 module "dns" {
   source           = "git::https://github.com/cloudposse/terraform-aws-route53-alias.git?ref=tags/0.2.5"
-  enabled          = "${var.enabled}"
+  enabled          = "${var.enabled ? var.dns_enabled : false}"
```
`enabled          = "${var.enabled == "true" && var.dns_enabled == "true" ? "true" : "false" }"`

ommadawn46 replied:
@joshmyers 
Thank you for review. I fixed it.

ommadawn46 commented:
@joshmyers 
Would you please review again?

aknysh commented on variables.tf:
```diff
@@ -275,3 +275,8 @@ variable "web_acl_id" {
+
+variable "dns_enabled" {
+  default     = "true"
+  description = "Select Disabled if you do not want to create DNS config after the distribution is created"
```
```suggestion
  description = "Set to `false` if you do not want to create the DNS hostname after the distribution is created"
```

ommadawn46 commented:
@aknysh Thank you for the review.

I changed to the same way as terraform-aws-rds-cluster, and deleted dns_record.
Without the following commit, it did not work, so I updated the module version.
https://github.com/cloudposse/terraform-aws-route53-alias/commit/74957ad27314622bc1d647f98216a1c8d6ee8130
